packages/components/src/view-more-list/README.md

```diff
@@ -0,0 +1,20 @@
+ViewMoreList
```

Comment:
This one also pops open under the WP menu

<img width="385" alt="Screen Shot 2019-09-06 at 11 44 03 AM" src="https://user-images.githubusercontent.com/343847/64436943-b6fc3100-d09b-11e9-8de7-c076d5e7e126.png">